reference/5.1/Microsoft.PowerShell.Management/Pop-Location.md

```diff
@@ -11,6 +11,7 @@ title: Pop-Location
 # Pop-Location
 
 ## SYNOPSIS
+

```

Comment:
Due to a bug in `Get-Help`, we need to remove the blank line after `## SYNOPSIS`. If there is a blank line, then `Get-Help` cannot display the Synopsis text.

Reply:
:) Done.